README.md

```diff
@@ -8,6 +8,15 @@ The [Cromwell documentation has a dedicated site](https://cromwell.readthedocs.i
 
 First time to Cromwell? Get started with [Tutorials](https://cromwell.readthedocs.io/en/stable/tutorials/FiveMinuteIntro/).
 
+### Google LifeSciences is being deprecated in July - migrate to GCP Batch
+
+Our WDL execution engine, Cromwell, currently leverages the Cloud Life Sciences API (sometimes referred to as PAPI) when launching workflows in Terra. 
```

Comment:
I think community users might not know of Terra and so maybe we can remove it here and in below lines?